DEV_TO_DO.md

```diff
@@ -3,7 +3,7 @@
 ## Monorepo
 
 - Get `nx format:check` and `nx format:write` to work like `prettier -c .` and `prettier -w .`
-  - Remove the custom scripts in `package.json`
+  - Remove the custom `format:` scripts in `package.json`
```

Comment:
FYI you may want to keep things as they are. I found a lot of chatter about the NX formatting having issues. I couldn't make it do something I wanted it to and ending up going back to an npm script.